components/FlashCard.js

```diff
@@ -7,18 +7,48 @@ import Button from "./Button";
 const StyledCard = styled.li`
   background-color: ${({ $showAnswer }) =>
     $showAnswer ? "#93E9BE" : "lightgray"};
+  position: relative;
   list-style: none;
   width: 80%;
   border-radius: 20px;
-  padding: 25px 25px;
+  padding: 25px 25px 80px;
   line-height: 1.25;
   &:hover {
     cursor: pointer;
   }
 `;
 
-export default function FlashCard({ card, onMarkCorrect, collections }) {
+const StyledDeleteContainer = styled.div`
+  position: absolute;
+  bottom: 10px;
+  right: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const StyledButtonContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 10px;
+`;
+
+const StyledWarning = styled.p`
+  background-color: #ffa500;
+  padding: 5px;
+  border-radius: 10px;
+  margin-bottom: 10px;
+`;
+
+export default function FlashCard({
+  card,
+  onMarkCorrect,
+  collections,
+  deleteCard,
+}) {
   const [showAnswer, setShowAnswer] = useState(false);
+  const [isClicked, setIsClicked] = useState(false);
```

Comment:
`isClicked` isn’t descriptive enough. This state specifically controls whether the delete prompt is visible, so let’s rename it to make its purpose clearer.

```suggestion
const [isDeletePromptVisible, setIsDeletePromptVisible] = useState(false);
```